components/wifi/esp8266-driver/ESP8266Interface.h

```diff
@@ -94,6 +98,9 @@ class ESP8266Interface : public NetworkStack, public WiFiInterface {
      *
      *  Attempts to connect to a WiFi network.
      *
+     *  If interface is configured blocking it will timeout after up to
+     *  ESP8266_INTERFACE_CONNECT_TIMEOUT_MS + ESP8266_CONNECT_TIMEOUT ms.
```

Comment:
_2 *_

Reply:
I think this "2 *" that you mean is already included within:
```
ESP8266_INTERFACE_CONNECT_TIMEOUT_MS (2 * ESP8266_CONNECT_TIMEOUT + ESP8266_INTERFACE_CONNECT_INTERVAL_MS)
```
We give ESP chances to retry for up to ESP8266_INTERFACE_CONNECT_TIMEOUT_MS (35 s), polling it every 5 s and then we give it another, very last chance (but without an interval) which may take up to ESP8266_CONNECT_TIMEOUT (15 s).
Is there something I am missing? :)

Reply:
Sorry, my mistake. Now everything seem to be fine.